gc/startup/omrgcalloc.cpp

```diff
@@ -60,18 +60,31 @@ allocHelper(OMR_VMThread * omrVMThread, size_t sizeInBytes, uintptr_t flags, boo
 
 	omrobjectptr_t heapBytes = (omrobjectptr_t)env->_objectAllocationInterface->allocateObject(env, &allocdescription, env->getMemorySpace(), collectOnFailure);
 
-	/* OMRTODO: Should we use zero TLH instead of memset? */
 	if (NULL != heapBytes) {
 		if (J9_ARE_ALL_BITS_SET(flags, OMR_GC_ALLOCATE_ZERO_MEMORY)) {
-			uintptr_t size = allocdescription.getBytesRequested();
-			memset(heapBytes, 0, size);
+			memset(heapBytes, 0, allocdescription.getBytesRequested());
 		}
+		/* TODO: Provide a generic object initialization model interface to be implemented for each language context. */
+#if defined(OMR_EXAMPLE)
```

Comment:
We won't, as soon as the TODO  on line 67 is done in my allocation model commit. This should stay in for now.